core/controllers/learner_group.py

```diff
@@ -314,6 +314,74 @@ def get(self, learner_group_id):
         self.render_json(all_learners_progress)
 
 
+class LearnerGroupLearnerSpecificProgressHandler(base.BaseHandler):
+    """Handles operations related to fetching learner specific progress."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    URL_PATH_ARGS_SCHEMAS = {
+        'learner_group_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.LEARNER_GROUP_ID_REGEX
+                }]
+            },
+            'default_value': None
+        }
+    }
+
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {}
+    }
+
+    @acl_decorators.can_access_learner_groups
+    def get(self, learner_group_id):
```

Comment:
Missing python typing for `learner_group_id` arg.

Reply:
Added